images/Dockerfile

```
@@ -41,6 +41,9 @@ ENV ImageOS=ubuntu22
 
 RUN apt-get update -y \
     && apt-get install -y --no-install-recommends \
+    jq \
+    git \
+    curl \
```

Can we consider adding `unzip`? Found that used in many actions from the marketplace.

```suggestion
    curl \
    unzip \
```